src/components/common/Social/Social.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+    FaFacebookF, FaTwitter, FaGithub, FaInstagram,
```

Comment:
Só a título de legibilidade:
```suggestion
   FaFacebookF as IconFacebook, FaTwitter as IconTwitter, FaGithub as IconGithub, FaInstagram as IconInstagram,
```